anaconda-pkg-build/linux/cuda-rocky8/Dockerfile

```diff
@@ -0,0 +1,120 @@
+# This container has the CUDA compilers installed. For versions less than 12, they're used,
+# because defaults' cudatoolkit <= 11.8 package doesn't provide compilers.
+# But not for versions equal and greater than 12.
+ARG CUDA_VERSION=12.4.1
+ARG GCC_VER=11.2.0
+FROM nvidia/cuda:${CUDA_VERSION}-devel-rockylinux8
+
+# this is mostly copied from public.ecr.aws/y0o4y9o3/anaconda-pkg-build
+
+# ulimit needs to be turned down: https://bugzilla.redhat.com/show_bug.cgi?id=1537564
+# hadolint ignore=DL3031,DL3033,SC3045
+RUN ulimit -n 1024 \
+    # Hack to force locale generation, if needed
+    && yum update -q -y glibc-common \
+    && yum install -q -y \
+        #----------------------------------------
+        # X11-related libraries needed for various CDTs
+        #----------------------------------------
+        libX11 \
+        libXau \
+        libxcb \
+        libXcomposite \
+        libXcursor \
+        libXdamage \
+        libXdmcp \
+        libXext \
+        libXfixes \
+        libXi \
+        libXinerama \
+        libXrandr \
+        libXrender \
+        libXScrnSaver \
+        libXt \
+        libXtst \
+        #----------------------------------------
+        # MESA 3D graphics library
+        #----------------------------------------
+        #mesa-libEGL \
+        #mesa-libGL \
+        #mesa-libGLU \
+        #----------------------------------------
+        # Vendor-neutral OpenGL
+        #----------------------------------------
+        libglvnd-opengl \
+        #----------------------------------------
+        # X11 virtual framebuffer; useful for testing GUI apps
+        #----------------------------------------
+        xorg-x11-server-Xvfb \
+        #----------------------------------------
+        # Other hardware and low-level system libraries
+        #----------------------------------------
+        #alsa-lib \
+        #libselinux \
+        #pam \
+        #pciutils-libs \
+        #----------------------------------------
+        # Low-level and basic system utilities.
+        #
+        # NOTE: previous versions of this image included tools like `patch`
+        # and `make`; these days, we prefer package recipes list the
+        # equivalent conda packages as build dependencies, rather than
+        # assume the build container provides these tools.
+        #----------------------------------------
+        curl \
+        file \
+        net-tools \
+        openssh-clients \
+        procps-ng \
+        psmisc \
+        rsync \
+        tar \
+        util-linux \
+        diffutils \
+        #wget \
+        which \
+    && yum clean all
+
+# Set the locale
+ENV LANG=en_US.UTF-8
+ENV LANGUAGE=en_US:en
+ENV LC_ALL=en_US.UTF-8
+
+# renovate: datasource=custom.miniconda_installer depName=Linux-x86_64.sh
+ARG INSTALLER_VERSION=py312_24.5.0-0
```

Comment:
You are requesting an older Miniconda version, but also instruct `renovate` to keep it updated. Please change `INSTALLER_VERSION` to the latest version or remove the `renovate` comment.

Reply:
Thanks. I removed the renovate comment - it shouldn't be renovated. If it is, conda-build v24.1.2 won't be able to be installed, which will make this image not usable in Anaconda's current CI (for reasons)